zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -652,7 +652,10 @@ private void updateAngularObjectRegistry(NotebookSocket conn, Note note) {
       try {
         interpreterGroup = findInterpreterGroupForParagraph(note, paragraph.getId());
       } catch (Exception e) {
-        e.printStackTrace();
+        LOG.error(e.getMessage(), e);
```

Comment:
this might not be an error? I think the a paragraph can be without a `%repl` in which case interpreterGroup can be null (which is valid)

consider LOG.warn

Reply:
ok.